test/gurka/test_use.cc

```diff
@@ -90,6 +90,7 @@ TEST_F(Use, test_passing_rest_area) {
   EXPECT_FALSE(steps[step_idx]["intersections"][0].HasMember("rest_stop"));
   EXPECT_TRUE(steps[step_idx]["intersections"][1].HasMember("rest_stop"));
   EXPECT_STREQ(steps[step_idx]["intersections"][1]["rest_stop"]["type"].GetString(), "rest_area");
+  EXPECT_STREQ(steps[step_idx]["intersections"][1]["rest_stop"]["name"].GetString(), "Bear Peak");
```

Comment:
The one line of test code that proves things work.  (I also thought I'd give the rest-area a name better than "Rest Area".)